services/socketService.js

```diff
@@ -52,21 +53,18 @@ module.exports.init = function (socketIo) {
       console.log('currently connected: ' + Object.keys(currentlyConnected).length)
     })
 
-    clientSocket.on('sessionDisconnected', function (data) { console.log('sessionDisconnected') })
+    clientSocket.on('sessionDisconnected', function (data) { cdr.addConnectionDestroyEvent(data) })
```

Comment:
This should call addSessionDisconnectionEvent(data). I think this was unclear on my part. Sorry about that!